docs/libraries/azure-core/rules/use-standard-integer.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "use-standard-integer"
+---
+
+```text title="Full name"
+@azure-tools/typespec-azure-core/use-standard-integer
+```
+
+Azure services favor the standard types int32 and int64 (or safeint) when generating code using Autorest.
+
+#### ❌ Incorrect
+
+```tsp
+model Widget {
+  id: int8;
+}
+```
```

Comment:
I think we should have an example that uses int32 and an example that uses safeInt